internal/credential/attributes.go

```diff
@@ -9,7 +9,7 @@ import (
 	"github.com/aws/aws-sdk-go/aws/endpoints"
 	"github.com/hashicorp/boundary-plugin-aws/internal/errors"
 	"github.com/hashicorp/boundary-plugin-aws/internal/values"
-	"github.com/hashicorp/go-secure-stdlib/awsutil"
+	awsutilv2 "github.com/hashicorp/go-secure-stdlib/awsutil/v2"
```

Comment:
Why add the import alias `awsutilv2` here? I would personally drop it and just use `awsutil` everywhere

Reply:
I changed the import alias to awsutil